apps/projects/helpers.py

```diff
@@ -41,3 +43,12 @@ def get_num_reported_unread_comments(project):
         .filter(num_reports__gt=0)
         .count()
     )
+
+
+def get_pagination(obj: object, project: object, per_page=15) -> int:
+    objects = obj.__class__.objects.filter(module=project)
+    p = Paginator(objects, per_page)
```

Comment:
tbh I don't think we need the `Paginator` here since you are computing the page number yourself. All you need is the index of `obj` in the `objects` query set. The `p.object_list` is literally just the objects query set you put in. You could equally simply iterate yourself:
```python
index = None
for i, other_object in enumerate(objects):
    if other_object == obj:
        index = i + 1

if index is None:
    # error case, obj not in objects
    page_number = 1
else:
    page_number = ceil(index / per_page)
```